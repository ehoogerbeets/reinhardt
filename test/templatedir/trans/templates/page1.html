{% blocktrans %}
This is text inside of a block trans tag. It is very long and treacherous
and it spans multiple
lines in the source.
{% endblocktrans %}
