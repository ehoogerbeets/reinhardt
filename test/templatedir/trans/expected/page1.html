 Dies ist Text innerhalb eines trans-Blocks. Es ist sehr lang und tückisch und es spannt mehrere Zeilen in der Quelle über. 
